101-default-shared-dashboard/azuredeploy.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
format the json, e.g. SHIFT+ALT+F in VS Code

Reply:
done